# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mesa-demos/hotfix-glx-pc.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mesa-demos-9.0.0/meson.build.vanilla	2023-12-01 15:38:39.093118899 +0100
+++ mesa-demos-9.0.0/meson.build	2023-12-01 15:39:37.623120434 +0100
@@ -98,7 +98,7 @@
     required : dep_x11.found())
 endif
 
-dep_glx = dependency('glx', required: false, disabler : true)
+dep_glx = dependency('gl', required: false, disabler : true)
 if not dep_glx.found() and host_machine.system() == 'darwin'
   # xquartz doesn't have a glx.pc, but it does have the header. And all the
   # symbols reside in libGL, so let's just use that.
